l10n_ch_qr_bill_scan/wizard/account_invoice_import.py

```diff
@@ -157,24 +157,3 @@ def parse_pdf_invoice(self, file_data):
                 return self.parse_qrbill(decoded_data)
             logger.debug("No Swiss QR-Code found in PDF file")
         return super().parse_pdf_invoice(file_data)
-
-    def _hook_no_partner_found(self, partner_dict):
```

Comment:
Since we require invoice import configuration on the partner, the user needs to specify it manually.

Reply:
can you tell more about that?

Reply:
When a user imports an invoice that doesn't have a partner, I assumed that [_hook_no_partner_found(v16 goto_no_partner(....)](https://github.com/OCA/edi/pull/753/files#diff-1caa8dd9f3d951855fdc8c448a6c62e2df706b345bd39a198b59b9043096a178)) would automatically create the missing partner (the 'Continue' button proceeds with the invoice import after creating the partner). However, that auto-created partner does not exist and the unset the 'Invoice Import Configuration' resulting in the inability to process the imported invoice further. 
User must manually create the partner and set the invoice import configuration then come back to import it again, so the 'Continue' button to have any effect and I removed them, if user must re-import, so the wizard updating kinda meaningless, so remove them too